Common_Engine/Query/IsSelfIntersecting.cs

```diff
@@ -0,0 +1,63 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2018, the respective contributors. All rights reserved.
```

Comment:
Check the year here. Should be 2019 in new/modified files

Reply:
Fixed